src/database.js

```diff
@@ -0,0 +1,33 @@
+import { MongoClient } from 'mongodb';
+
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const { DB_USER, DB_PASSWORD } = process.env;
+
+const MONGO_URL = `mongodb://${DB_USER}:${DB_PASSWORD}@ds227171.mlab.com:27171/theremotants`;
```

Comment:
@Saumya29 instead of fetching the user and password from the env, fetch the mongo uri and the db-name.

Reply:
Done!